src/api/core/abstract/base.api.ts

```diff
@@ -4,9 +4,13 @@ import type { BaseApiRequest } from '../types';
 export abstract class BaseApi {
   createRequest = <T>({ mock, request }: BaseApiRequest): Promise<T> => {
     if (isMockEnv && mock) {
-      return mock().then(
-        (module: { readonly default: Promise<T> }) => module.default
-      );
+      return mock().then((module: { readonly default: Promise<T> }) => {
+        if (typeof module.default !== 'undefined') {
+          return module.default;
+        }
+
+        return module;
+      });
```

Comment:
В общем, здесь в модуле надо вернуть функцию, которая возвращает промис, затем во втором `then` этот промис вызывается:

```suggestion
      return mock()
        .then(
          (module: { readonly default: () => Promise<T> }) => module.default
        )
        .then((fn) => fn());
```